### PR TITLE
Snapcraft installation instructions added

### PR DESCRIPTION
**Pull request by idvoretskyi**
Helm is available as a snap - https://snapcraft.io/helm; added this to the installation options.

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ If you want to use a package manager:
 - [Chocolatey](https://chocolatey.org/) users can use `choco install kubernetes-helm`.
 - [Scoop](https://scoop.sh/) users can use `scoop install helm`.
 - [GoFish](https://gofi.sh/) users can use `gofish install helm`.
+- [Snapcraft](https://snapcraft.io/) users can use `snap install helm --classic`
```
Comment (hickeyma) [inline README.md]: This probably should this be: `snap install helm`

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for adding this @idvoretskyi . Small change. Also, PR to update the doc is needed: https://github.com/helm/helm-www/blob/master/content/en/docs/intro/install.md

File: README.md (hunk 1 above)
Comment (bacongobbler) [reply README.md]: This appears correct according to https://github.com/snapcrafters/helm/blob/master/README.md

I believe the flag is intentional to handle a case in snap’s configuration. Not sure of the reason behind it, but that flag is required to install helm last time I tried.

Comment (idvoretskyi) [reply README.md]: @bacongobbler is correct, this snap requires the `--classic` flag for the "classic" confinement. You may find more details here - https://snapcraft.io/docs/snap-confinement.

PS. It also requires the superuser power to install, but I haven't added `sudo` following the consistency with the installation methods listed above.

**Review by bacongobbler — APPROVED** [review]

**Comment by idvoretskyi** [comment]
@hickeyma submitted another PR to the website repo - https://github.com/helm/helm-www/pull/550, PTAL.

@bacongobbler thanks :)

File: README.md (hunk 1 above)
Comment (hickeyma) [reply README.md]: OK, my bad, thought it was a typo

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @idvoretskyi

**Pull request merged**